modules/sdk-coin-avaxp/src/lib/transaction.ts

```diff
@@ -258,14 +258,37 @@ export class Transaction extends BaseTransaction {
   /** @inheritdoc */
   explainTransaction(): TransactionExplanation {
     const txJson = this.toJson();
-    const displayOrder = ['id', 'outputAmount', 'changeAmount', 'outputs', 'changeOutputs', 'fee', 'type'];
+    const displayOrder = [
+      'id',
+      'outputAmount',
+      'changeAmount',
+      'outputs',
+      'changeOutputs',
+      'rewardAddresses',
+      'fee',
+      'type',
+    ];
+
+    let totalIn = new BN(0);
+    this._utxos.map((utxo) => {
+      totalIn = totalIn.add(new BN(utxo.amount));
+    });
+    const outputAmount = txJson.outputs.reduce((p, n) => p.add(new BN(n.value)), new BN(0)).toString();
+    const changeAmount = totalIn.sub(new BN(outputAmount)).toString();
```

Comment:
Is this correct? Is the change amount implicit? Usually this is how the mining fee is calculated for utxo coins and not the change amount, which appears as explicit output(s) on the transaction.

Reply:
There are [no network fees for delegating or validating for avalanche](https://docs.avax.network/quickstart/transaction-fees#fee-schedule), which are the only two things that we do with the P-chain. So the only thing contributing to the outputs are the amount that you are staking and the change left over from your original amount that gets sent back as change

Reply:
and that change amount is not an output on the transaction? It's left implicit as the difference between the input and output amounts?

Reply:
I see this alternatives:
1.- set the change amount to 0, as an account. This is implicit now.
2.- copy utxo behavior. See it below

https://github.com/BitGo/BitGoJS/blob/fedcb7e6c4c4078cc1425f74f068f6933a64861b/modules/core/src/v2/coins/abstractUtxoCoin.ts#L1256-L1289